### ISSUE TITLE
 level command produces Invisible blocks when trying to Specify a Block Type to use

### ISSUE THREAD

**Issue by Conchipedia** [StrangeLoopGames/EcoIssues#19205]
\
**Version:** 0.9.0.5 beta release-100

**/level 9,9** works as intended with default grass blocks being the bottom level
**/level 9,9,WaterBlock** levels area with bottom level being water (as expected)
**/level 9,9,EmptyBlock**  levels area with nothing (as expected)

But trying with things like **/level 9,9,MortaredSandstone** will make the area flat but will leave invisible blocks 
Tested that blocks are actually there by trying to place down a workbench and it says **Can't place there blocked**
also placing dirt above the invisible blocks will just let me place dirt on **top** of the invisible blocks.
![snap7577](https://user-images.githubusercontent.com/29383891/98468250-906ee400-21d1-11eb-97c6-2f97ef194091.png)

![snap7576](https://user-images.githubusercontent.com/29383891/98468230-70d7bb80-21d1-11eb-9933-729d0e376fa6.png)

**Comment by SlayksWood** [comment]
@Conchipedia  What result do you expect? Empty blocks appear because you have spammed a block that is not present. To spawn constructed block with this command, you should add FormType, for example /level 9,9,MortaredSandstoneFloor.

**Comment by Conchipedia** [comment]
i thought it used the **default** Block type like it always used to do, which was a Solid cube

**Comment by Conchipedia** [comment]
if a **Formtype** is left off the end of a constructed block, it would be nice to say something along the lines 
**please add formtype**  rather than just allowing invisible blocks

**Comment by IcedForge** [comment]
I agree with Conchipedia in the fact that it should not allow to place blocks that has no form functionality OR it should add basic cube blocks instead as a placeholder instead of invisible incorrect blocks.
I mentioned this issue in #17723 
Also discussed in #14627
